common/app/common/dfp/TagSponsorship.scala

```diff
@@ -97,11 +98,9 @@ case class HighMerchandisingLineItem(
 
     lazy val matchesEdition: Boolean = editions.isEmpty || editions.exists(_.contains(cleansedPageEdition))
 
-    if(matchesTag && matchesAdUnit && matchesEdition){
-      println(name)
-    }
+    lazy val matchesUrl: Boolean = urls.isEmpty || urls.exists(_.contains(url))
 
-    matchesTag && matchesAdUnit && matchesEdition
+    matchesTag && matchesAdUnit && matchesEdition && matchesUrl
```

Comment:
@kelvin-chappell @kenlim just added this final check for url if set because now some (a couple) line items dont have adunit target OR tags target but do have 'high-merchandising' slot target, so are causing rendering of the slot on every page, this is just one more check.